src/lib/pickles/impls.mli

```diff
@@ -130,7 +130,10 @@ module Wrap : sig
   end
 
   val input :
-       unit
+       feature_flags:
+         Pickles_types.Plonk_types.Opt.Flag.t
+         Pickles_types.Plonk_types.Features.t
```

Comment:
```suggestion
         Pickles_types.Plonk_types.Features.options
```
Maybe the names that I introduced (`options` / `flags`) are not appropriate or misleading